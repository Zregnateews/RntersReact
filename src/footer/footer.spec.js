import React from 'react';
import { render } from '@testing-library/react';
import { Footer } from './footer';

describe('Footer', () => {
  it('renders correctly', () => {
    const wrapper = render(<Footer />);
    expect(wrapper).toMatchSnapshot();
  });
})
