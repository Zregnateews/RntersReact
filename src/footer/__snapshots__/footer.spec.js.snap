// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders correctly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <footer
        class="not_visible_mobile"
      >
        <div
          class="container-fluid"
        >
          <div
            class="row"
          >
            <div
              class="col-sm-4 footer_section"
            >
              <p>
                <b>
                  LINKS ÚTEIS
                </b>
              </p>
              <p>
                <a
                  href="/pt/pages/security"
                >
                  Garantia SafeStuff
                </a>
              </p>
              <p>
                <a
                  href="/pt/pages/faq"
                >
                  FAQ
                </a>
              </p>
              <p>
                <a
                  href="/pt/pages/terms"
                >
                  Termos de Serviço
                </a>
              </p>
              <p>
                <a
                  href="/pt/pages/privacy"
                >
                  Política de Privacidade
                </a>
              </p>
            </div>
            <div
              class="col-sm-4 footer_section text-left"
            >
              <p>
                <b>
                  CONNECT
                </b>
              </p>
              <p
                class="footer__subscribe_text"
              >
                Não percas as novidades!
              </p>
              <form
                accept-charset="UTF-8"
                action="/pt/subscribe_newsletter"
                class="footer__subscribe_form form-inline"
                method="post"
              >
                <input
                  name="utf8"
                  type="hidden"
                  value="✓"
                />
                <input
                  name="authenticity_token"
                  type="hidden"
                  value="pj0JuSmlvEehsiycv4Jpmnz80EhkXOAtf5B+F5W1lYna7Q+AH57LBflwQ59FpaNSm+qyQGVYlpyO3eWhoP5dlw=="
                />
                <div
                  class="row"
                  style="margin-bottom: 10px;"
                >
                  <div
                    class="col-xs-6"
                  >
                    <input
                      class="form-control"
                      id="_email"
                      name="[email]"
                      placeholder="O teu email"
                      required=""
                      type="email"
                    />
                  </div>
                  <div
                    class="col-xs-6"
                  >
                    <input
                      class="btn btn-primary"
                      name="commit"
                      type="submit"
                      value="Subscreve"
                    />
                  </div>
                </div>
              </form>
              <p>
                <a
                  class="social_icons"
                  href="https://www.facebook.com/Rnters-452745608263925"
                >
                  <i
                    class="fab fa-facebook-f fa-lg"
                  />
                   Like and share
                </a>
              </p>
              <p>
                <a
                  class="social_icons"
                  href="https://www.instagram.com/rnters.pt/"
                >
                  <i
                    class="fab fa-instagram fa-lg"
                  />
                   Follow us
                </a>
              </p>
            </div>
            <div
              class="col-sm-4 footer_section"
            >
              <p>
                <b
                  style="text-transform: uppercase;"
                >
                  Para mais informações contacta:
                </b>
              </p>
              <div
                class="row"
              >
                <div
                  class="col-sm-2"
                >
                  <img
                    alt="Logo brand primary"
                    src="https://d3urocn5qjrv7i.cloudfront.net/assets/logo_brand_primary-576ad0b1b49e8bde9dd8c6ffa1ae8384c65f3fb696589c69897463951fcab743.png"
                    style="height: 50px;"
                  />
                </div>
                <div
                  class="col-sm-10"
                >
                  <p>
                    <a
                      data-turbolinks="false"
                      href="/en/items/1960-porto-portugal-alugar-babolat-pure-drive-raquete-tenis?change_locale=true"
                    >
                      English
                    </a>
                    |
                    <a
                      data-turbolinks="false"
                      href="/pt/items/1960-porto-portugal-alugar-babolat-pure-drive-raquete-tenis?change_locale=true"
                    >
                      Português
                    </a>
                  </p>
                  <p>
                    <a
                      class="social_icons"
                      href="mailto:info@rnters.com"
                    >
                      <i
                        class="fas fa-envelope fa-lg"
                      />
                      info@rnters.com
                    </a>
                  </p>
                  <p>
                    2016-2020 Rnters ©
                  </p>
                </div>
              </div>
            </div>
          </div>
        </div>
      </footer>
    </div>
  </body>,
  "container": <div>
    <footer
      class="not_visible_mobile"
    >
      <div
        class="container-fluid"
      >
        <div
          class="row"
        >
          <div
            class="col-sm-4 footer_section"
          >
            <p>
              <b>
                LINKS ÚTEIS
              </b>
            </p>
            <p>
              <a
                href="/pt/pages/security"
              >
                Garantia SafeStuff
              </a>
            </p>
            <p>
              <a
                href="/pt/pages/faq"
              >
                FAQ
              </a>
            </p>
            <p>
              <a
                href="/pt/pages/terms"
              >
                Termos de Serviço
              </a>
            </p>
            <p>
              <a
                href="/pt/pages/privacy"
              >
                Política de Privacidade
              </a>
            </p>
          </div>
          <div
            class="col-sm-4 footer_section text-left"
          >
            <p>
              <b>
                CONNECT
              </b>
            </p>
            <p
              class="footer__subscribe_text"
            >
              Não percas as novidades!
            </p>
            <form
              accept-charset="UTF-8"
              action="/pt/subscribe_newsletter"
              class="footer__subscribe_form form-inline"
              method="post"
            >
              <input
                name="utf8"
                type="hidden"
                value="✓"
              />
              <input
                name="authenticity_token"
                type="hidden"
                value="pj0JuSmlvEehsiycv4Jpmnz80EhkXOAtf5B+F5W1lYna7Q+AH57LBflwQ59FpaNSm+qyQGVYlpyO3eWhoP5dlw=="
              />
              <div
                class="row"
                style="margin-bottom: 10px;"
              >
                <div
                  class="col-xs-6"
                >
                  <input
                    class="form-control"
                    id="_email"
                    name="[email]"
                    placeholder="O teu email"
                    required=""
                    type="email"
                  />
                </div>
                <div
                  class="col-xs-6"
                >
                  <input
                    class="btn btn-primary"
                    name="commit"
                    type="submit"
                    value="Subscreve"
                  />
                </div>
              </div>
            </form>
            <p>
              <a
                class="social_icons"
                href="https://www.facebook.com/Rnters-452745608263925"
              >
                <i
                  class="fab fa-facebook-f fa-lg"
                />
                 Like and share
              </a>
            </p>
            <p>
              <a
                class="social_icons"
                href="https://www.instagram.com/rnters.pt/"
              >
                <i
                  class="fab fa-instagram fa-lg"
                />
                 Follow us
              </a>
            </p>
          </div>
          <div
            class="col-sm-4 footer_section"
          >
            <p>
              <b
                style="text-transform: uppercase;"
              >
                Para mais informações contacta:
              </b>
            </p>
            <div
              class="row"
            >
              <div
                class="col-sm-2"
              >
                <img
                  alt="Logo brand primary"
                  src="https://d3urocn5qjrv7i.cloudfront.net/assets/logo_brand_primary-576ad0b1b49e8bde9dd8c6ffa1ae8384c65f3fb696589c69897463951fcab743.png"
                  style="height: 50px;"
                />
              </div>
              <div
                class="col-sm-10"
              >
                <p>
                  <a
                    data-turbolinks="false"
                    href="/en/items/1960-porto-portugal-alugar-babolat-pure-drive-raquete-tenis?change_locale=true"
                  >
                    English
                  </a>
                  |
                  <a
                    data-turbolinks="false"
                    href="/pt/items/1960-porto-portugal-alugar-babolat-pure-drive-raquete-tenis?change_locale=true"
                  >
                    Português
                  </a>
                </p>
                <p>
                  <a
                    class="social_icons"
                    href="mailto:info@rnters.com"
                  >
                    <i
                      class="fas fa-envelope fa-lg"
                    />
                    info@rnters.com
                  </a>
                </p>
                <p>
                  2016-2020 Rnters ©
                </p>
              </div>
            </div>
          </div>
        </div>
      </div>
    </footer>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
